src/Common/tests/Collections/IEnumerableTest.cs

```diff
@@ -0,0 +1,428 @@
+        {
+            RepeatTest((enumerator, items) =>
+                       {
+                           while (enumerator.MoveNext())
+                           {
+                           }
+
+                           Assert.False(enumerator.MoveNext());
+                       });
+        }
+
+        [Fact]
+        public void Current()
+        {
+            RepeatTest((enumerator, items, iteration) =>
+                       {
+                           if (iteration == 1)
+                               VerifyEnumerator(items, enumerator, 0, items.Length / 2, false, true);
+                           else
+                               VerifyEnumerator(items, enumerator);
+                       });
+        }
+
+        [Fact]
+        public void Reset()
+        {
+            if (IsResetNotSupported)
+            {
+                RepeatTest((enumerator, items) =>
+                           {
+                               Assert.Throws<NotSupportedException>(
+                                   () => enumerator.Reset());
+                           });
+                RepeatTest((enumerator, items, iter) =>
+                           {
+                               if (iter == 1)
+                               {
+                                   VerifyEnumerator(items, enumerator, 0, items.Length/2, false, true);
+                                   for (int i = 0; i < 3; i++)
+                                   {
+                                       Assert.Throws<NotSupportedException>(
+                                           () => enumerator.Reset());
+                                   }
+                                   VerifyEnumerator(
+                                       items,
+                                       enumerator,
+                                       items.Length/2,
+                                       items.Length - (items.Length/2),
+                                       true,
+                                       false);
+                               }
+                               else if (iter == 2)
+                               {
+                                   VerifyEnumerator(items, enumerator);
+                                   for (int i = 0; i < 3; i++)
+                                   {
+                                       Assert.Throws<NotSupportedException>(
+                                           () => enumerator.Reset());
+                                   }
+                                   VerifyEnumerator(items, enumerator, 0, 0, true, false);
+                               }
+                               else
+                               {
+                                   VerifyEnumerator(items, enumerator);
+                               }
+                           });
+            }
+            else
+            {
+                RepeatTest((enumerator, items, iter) =>
+                           {
```

Comment:
While I can't find what our rule is on alignment for lambdas, I'd be curious to see what we made the code formatter do.